src/Gotovka/FryCookExtendedViaInheritance.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: rokka
+ * Date: 31.10.16
+ * Time: 2:10
+ */
+
+namespace Vendor\Reporter\Gotovka;
+
+
+class FryCookExtendedViaInheritance extends FryCook
+{
+    public function __construct($extraDish)
+    {
+        $dishName = $extraDish;
```

Comment:
$dishName is a local variable, it will dissappear after the constructor finishes. Maybe you wanted to have some private or protected property in this class to store $extraDish?